app/controllers/pages_controller.rb

```diff
@@ -2,6 +2,7 @@ class PagesController < ApplicationController
   def index
     @flight = Flight.new
     @flights = Flight.order("RANDOM()").limit(10)
+    @seats = 1
     @airports = Airport.all
```

Comment:
When you have too many instance variables in an action, you can use presenters
You can check this out `https://github.com/justin808/fat-code-refactoring-techniques/pull/11`